pkgs/development/python-modules/imbalanced-learn/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, buildPythonPackage, fetchPypi, scikitlearn, pandas, nose, pytest }:
+
+buildPythonPackage rec {
+
+    pname = "imbalanced-learn";
+    version = "0.3.3";
+
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "1r5js9kw6rvmfvxxkfjlcxv5xn5h19qvg7d41byilxwq9kd515g4";
+    };
+
+    preConfigure = ''
+      export HOME=$PWD
+    '';
+
+    propagatedBuildInputs = [ scikitlearn ];
+    buildInputs = [ pandas ];
```

Comment:
Do you know if this is an optional dependency? Why is it in `buildInputs`?

Reply:
Looking at the source code, it is optional and only used in a test utility. Without it, a number of tests are skipped. Moved to `checkInputs`.